containers/ecr-viewer/src/app/components/Header.tsx

```diff
@@ -1,34 +1,40 @@
-import React from "react";
+"use client";
 
-import Link from "next/link";
+import React, { ReactNode } from "react";
 
-import { SignOutButton } from "./SignOutButton";
+import Image from "next/image";
+import Link from "next/link";
 
 /**
  * Header component for the ECR Viewer project.
  * This component renders the header section of the application, including the
  * navigation container, navbar, and logo. It uses USWDS (U.S. Web Design System)
- * classes for styling.
+ * classes for styling with some customization.
+ * @param props - Component props.
+ * @param props.children - nav links to be displayed in the header
  * @returns The header section of the application.
  */
-const Header: React.FC = () => (
-  <header className="usa-header usa-header--basic">
-    <div
-      className="usa-nav-container flex-row flex-align-center flex-justify"
-      style={{ maxWidth: "none" }}
-    >
-      <div className="usa-navbar">
-        <div className="usa-logo">
-          <h1>
-            <Link href="/" prefetch={false}>
-              DIBBs eCR Viewer
-            </Link>
-          </h1>
-        </div>
+const Header = ({ children }: { children: ReactNode }) => {
+  return (
+    <header className="usa-header usa-header--basic position-relative z-top isolate">
+      <div
+        className="usa-nav-container display-flex flex-direction-row"
+        style={{ maxWidth: "none" }}
+      >
+        <h1>
+          <Link className="dibbs-logo" href="/" prefetch={false}>
+            <Image
+              src="/ecr-viewer/ecr-viewer-logo.png"
+              alt="eCR Viewer Logo"
+              width={185}
+              height={40}
+            />
+          </Link>
```

Comment:
it feels weird to have the only contents of an h1 be an image - can we instead have the image-y part be an image and make the text just text. The text also appears a bit fuzzy on my screen, so it'd be crisper to have it outside the image and then then the h1 also makes more sense